### PR TITLE
add how to get help content

### PR DESCRIPTION
**Pull request by simonschwartz**

### CODE REVIEW

File: docs/usage/lodge_support_ticket.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## I need help using cloud.gov.au
+
+We use Zendesk to manage support tickets for cloud.gov.au. This service is available for anyone with a .gov.au email address.
+
+Before you lodge a support ticket be sure to check our documentation. You can also check the [status of cloud.gov.au](http://status.cloud.gov.au/) if you are experiencing an outage.
+
+<a href="https://digital89.zendesk.com/" role="button">Lodge a support ticket</a>
```
Comment (tessereth) [inline docs/usage/lodge_support_ticket.md]: Testing with someone who has a zendesk account but isn't an agent, going to this address they had to:
1. Sign in (can sign in with google)
2. Find the submit feedback link (hiding in the top bar)

It's probably fine for now but might require improving later.

File: docs/usage/lodge_support_ticket.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## I want to give feedback
+
+- If you are a member of the Digital Transformation Agency Slack you can talk to us in #cloud-gov-au
+
+### Feedback on documentation
+
+- [Raise a GitHub issue](https://github.com/AusDTO/cga_docs/issues)
+- You can propose edits to our documentation. Click the 'Edit this page on GitHub' at the bottom of any page.
```
Comment (tessereth) [inline docs/usage/lodge_support_ticket.md:14]: Is it worth linking to the contributing guide?

**Review by tessereth — COMMENTED** [review]
Looks good

File: docs/usage/lodge_support_ticket.md (hunk 1 above)
Comment (simonschwartz) [reply docs/usage/lodge_support_ticket.md]: Would a better UX be to have this as a mailto: ?

**Pull request merged**